Ginger/Ginger/App.xaml

```diff
@@ -9,7 +9,7 @@
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceDictionary Source="pack://application:,,,/Ginger;component/Dictionaries/Skins/GingerDefualtSkinDictionary.xaml"/>
-                <ResourceDictionary Source="pack://application:,,,/Ginger;component/Dictionaries/Terminologies/GingerDefualtTerminologyDictionary.xaml"/>
+                <!--<ResourceDictionary Source="pack://application:,,,/Ginger;component/Dictionaries/Terminologies/GingerDefualtTerminologyDictionary.xaml"/>-->
             </ResourceDictionary.MergedDictionaries>
```

Comment:
please remove commented out code, Codacy won't allow it

Reply:
Removed commented out code